src/components/InfoBanner/InfoBanner.tsx

```diff
@@ -106,21 +106,41 @@ const emphasizedIconColorVariants = styledVariant({
     }
 });
 
-const RoundedBox = styled(Box)<BoxWithVariant>`
+export const RoundedBox = styled(Box)<BoxWithVariant>`
     display: flex;
     flex-direction: row;
     justify-content: flex-start;
     border-radius: ${get('radii.3')};
     border: 0.0625rem solid;
     padding: ${`${Spaces[1]} ${Spaces[2]} ${Spaces[1]} ${Spaces[1]}`};
     ${({ emphasized }) => (emphasized ? emphasizedBannerVariants : bannerVariants)};
+
+    .Infobanner__Title,
```

Comment:
I propose to use the inversion of control here. Instead of cascading, we set CSS variables for colours. Building blocks use those colours, but they set their style:
<img width="3049" alt="image" src="https://github.com/freenowtech/wave/assets/1469636/59e8280c-48d9-4f80-805c-2cdb4a29bf47">

Reply:
Nice suggestion, I've done that 👍